packs/age-of-ashes-bestiary/nketiah.json

```diff
@@ -1997,7 +1997,7 @@
                     "die": "d4"
                 },
                 "description": {
-                    "value": "<p>This oak branch grows leaves in spring that change color in autumn and shed in winter. While wielding it, you gain a +2 circumstance bonus to Nature checks to identify plants.</p>\n<p><strong>Activate</strong> Cast a Spell</p>\n<hr />\n<p><strong>Effect</strong> You expend a number of charges from the staff to cast a spell from its list.</p>\n<ul>\n<li><strong>Cantrips</strong> <em>@UUID[Compendium.pf2e.spells-srd.Item.Tangle Vine]</em></li>\n<li><strong>1st</strong> <em>@UUID[Compendium.pf2e.spells-srd.Item.Shillelagh]</em> (can target the verdant staff)</li>\n<li><strong>2nd</strong> <em>@UUID[Compendium.pf2e.spells-srd.Item.Oaken Resilience]</em>, <em>@UUID[Compendium.pf2e.spells-srd.Item.Entangling Flora]</em>, <em>@UUID[Compendium.pf2e.spells-srd.Item.Shape Wood]</em>, <em>@UUID[Compendium.pf2e.spells-srd.Item.One with Plants]</em></li>\n</ul>\n<p><strong>Craft Requirements</strong> Supply one casting of all listed levels of all listed spells.</p>"
+                    "value": "<p>This oak branch grows leaves in spring that change color in autumn and shed in winter. While wielding it, you gain a +2 circumstance bonus to checks to identify plants and fungi.</p>\n<p><strong>Activate</strong> Cast a Spell</p>\n<hr />\n<p><strong>Effect</strong> You expend a number of charges from the staff to cast a spell from its list.</p>\n<ul>\n<li><strong>Cantrips</strong> <em>@UUID[Compendium.pf2e.spells-srd.Item.Tangle Vine]</em></li>\n<li><strong>1st</strong> <em>@UUID[Compendium.pf2e.spells-srd.Item.Runic Body], @UUID[Compendium.pf2e.spells-srd.Item.Runic Weapon]<br /></em></li>\n<li><strong>2nd</strong> <em>@UUID[Compendium.pf2e.spells-srd.Item.Entangling Flora], @UUID[Compendium.pf2e.spells-srd.Item.Oaken Resilience], @UUID[Compendium.pf2e.spells-srd.Item.One with Plants], @UUID[Compendium.pf2e.spells-srd.Item.Shape Wood]</em></li>\n</ul>\n<hr />\n<p><strong>Craft Requirements</strong> Supply one casting of all listed ranks of all listed spells.</p>"
```

Comment:
```suggestion
                    "value": "<p>This oak branch grows leaves in spring that change color in autumn and shed in winter. While wielding it, you gain a +2 circumstance bonus to checks to identify plants and fungi.</p>\n<p><strong>Activate</strong> Cast a Spell</p>\n<hr />\n<p><strong>Effect</strong> You expend a number of charges from the staff to cast a spell from its list.</p>\n<ul>\n<li><strong>Cantrips</strong> @UUID[Compendium.pf2e.spells-srd.Item.Tangle Vine]</li>\n<li><strong>1st</strong> @UUID[Compendium.pf2e.spells-srd.Item.Runic Body], @UUID[Compendium.pf2e.spells-srd.Item.Runic Weapon]</li>\n<li><strong>2nd</strong> @UUID[Compendium.pf2e.spells-srd.Item.Entangling Flora], @UUID[Compendium.pf2e.spells-srd.Item.Oaken Resilience], @UUID[Compendium.pf2e.spells-srd.Item.One with Plants], @UUID[Compendium.pf2e.spells-srd.Item.Shape Wood]</li>\n</ul>\n<hr />\n<p><strong>Craft Requirements</strong> Supply one casting of all listed ranks of all listed spells.</p>"
```

Reply:
Fair enough. I’ll update the compendium item as well.